subsys/logging/log_backend_rtt.c

```diff
@@ -108,18 +108,19 @@ static int char_out_drop_mode(u8_t data)
 
 static int line_out_drop_mode(void)
 {
-	*line_pos = '\r';
+	*(line_pos - 1) = '\r';
```

Comment:
If there is only `\n` in the buffer this can cause writing outside of an array this pointer is pointing to.

Reply:
handled